flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/plan/utils/PythonUtil.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.planner.plan.utils
+
+import org.apache.calcite.rex.{RexCall, RexNode}
+import org.apache.flink.table.functions.FunctionLanguage
+import org.apache.flink.table.planner.functions.utils.ScalarSqlFunction
+
+import scala.collection.JavaConversions._
+
+object PythonUtil {
+  /**
+    * Checks whether it contains the specified kind of function in a RexNode.
+    *
+    * @param expectedLanguage the expected kind of function to find
+    * @param recursive whether check the inputs
+    */
+  class FunctionFinder(expectedLanguage: FunctionLanguage, recursive: Boolean)
```

Comment:
There is a method `containsFunctionOf`, could you also port it to blink planner and also make `FunctionFinder` private?